projects/custody-key-dates-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/custody/date/KeyDate.kt

```diff
@@ -48,4 +42,6 @@ class KeyDate(
             it.version = version
         }
     }
+
+    fun getSoftDeleteFlag() = softDeleted
```

Comment:
Do you need this getter?

Reply:
i was using it in my test, but will delete it, i am can use the property directly

Reply:
done